packrat/lib/x86_64-pc-linux-gnu/3.6.3/covidImportation/DESCRIPTION

```diff
@@ -0,0 +1,36 @@
+Package: covidImportation
+Title: Estimating the Number of Imported COVID-19 Cases
+Version: 1.5
```

Comment:
hm... this looks wrong

Reply:
The importation tests are passing, so I guess this version is ok.

Reply:
can we change the importation package to the tag "v1.8"? I have updated to fix some errors that might have been causing issues.